### DISCUSSION TITLE
Lecture-02 Question

### DISCUSSION THREAD

**Discussion by SentientBiped** [cooperrc/advanced-dynamics#13] [General]
![image](https://github.com/cooperrc/advanced-dynamics/assets/157156914/9b6954af-bfdf-4cfd-b2d3-7466ae959a13)

For the final question in the above lecture, 'What is the maximum and minimum speed of point C in the example?' Can you solve this analytically? If so, how?

**Comment by cooperrc** [comment]
For $|v_C|$, the $\sin\theta_1$ and $\sin\theta_2$ can only be $-1...1$ so max x-magnitude is $\dot{x} + L_1\dot{\theta} + L_2\dot{\theta}$ 

and the max $\dot{\theta_i}$ values are $50\cdot 0.2\pi~rad/s$

If you calculate all the speeds of C as a function of time I think its within 1-2\% of this estimated magnitude.
